app/RootViewController.m

```diff
@@ -0,0 +1,70 @@
+#import "RootViewController.h"
+
+@interface RootViewController ()
+@end
+
+@implementation RootViewController
+
+- (void)loadView {
+  [super loadView];
+
+  self.view.backgroundColor = UIColor.blackColor;
+}
+
+- (void)viewDidLoad {
+  [super viewDidLoad];
+
+  _titleLabel =
+      [[UILabel alloc] initWithFrame:CGRectMake(0, 50, UIScreen.mainScreen.bounds.size.width, 100)];
+  _titleLabel.text = @"vnodebypass";
+  _titleLabel.textAlignment = NSTextAlignmentCenter;
+  _titleLabel.textColor = UIColor.whiteColor;
+  _titleLabel.font = [UIFont systemFontOfSize:40];
+  [self.view addSubview:_titleLabel];
+
+  _subtitleLabel = [[UILabel alloc]
+      initWithFrame:CGRectMake(0, 100, UIScreen.mainScreen.bounds.size.width, 100)];
+  _subtitleLabel.text = @"USE IT AT YOUR OWN RISK!";
+  _subtitleLabel.textAlignment = NSTextAlignmentCenter;
+  _subtitleLabel.textColor = UIColor.whiteColor;
+  _subtitleLabel.font = [UIFont systemFontOfSize:20];
+  [self.view addSubview:_subtitleLabel];
+
+  _button = [UIButton buttonWithType:UIButtonTypeSystem];
+  _button.frame = CGRectMake(UIScreen.mainScreen.bounds.size.width / 2 - 30,
+                             UIScreen.mainScreen.bounds.size.height / 2 - 25, 60, 50);
+  [_button setTitle:access("/bin/bash", F_OK) == 0 ? @"Enable" : @"Disable"
+           forState:UIControlStateNormal];
+  [_button addTarget:self
+                action:@selector(buttonPressed:)
+      forControlEvents:UIControlEventTouchUpInside];
+  [self.view addSubview:_button];
+}
+
+- (void)buttonPressed:(UIButton *)sender {
+  BOOL disabled = access("/bin/bash", F_OK) == 0;
```

Comment:
Please choose a different path that is common accross all jailbreaks to check, as bash doesn't come preinstalled on jailbreaks using Procursus. Procursus uses zsh as the default shell instead of bash, which means bash will only be installed if the user installs bash.